automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -229,6 +229,7 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     val testResult: Try[T] = Try {
       val cluster = createAndMonitor(googleProject, name, request)
       cluster.status shouldBe ClusterStatus.Error
+      cluster.errors should have size 1
```

Comment:
Anything else we can verify about the error?
- includes the expected error code
- message includes a path to the staging bucket
- timestamp is a valid timestamp

Reply:
Added check for error code and error message